docs/ddl-of-carbondata.md

```diff
@@ -805,7 +805,7 @@ Users can specify which columns to include and exclude for local dictionary gene
      2. If a column to be dropped has any Secondary index table created on them, drop column operation fails and the user will 
      be asked to drop the corresponding SI table first before going for actual drop.
 
-   - #### CHANGE COLUMN NAME/TYPE/COMMENT
+   - #### change-column-nametype
```

Comment:
this name doesn't look right, why changed? if changed also it should be name/type/comment right

Reply:
link was not directing to its description properly due to wrong name. This name was not chosen by me but was already there. Changing it to change-column-name-type-comment